ocaml/xapi/context.ml

```diff
@@ -277,8 +305,8 @@ let make_attributes ?task_name ?task_id ?task_uuid ?session_id ?origin () =
         match origin with
         | Internal ->
             [("xs.xapi.task.origin", "internal")]
-        | Http (req, _) ->
-            attr_of_req req
+        | Http (req, s) ->
```

Comment:
we may want a [("xs.xapi.task.origin", "http")], in opposition to the alreay existing one for "internal" ?